webapp/src/Entity/Contest.php

```diff
@@ -634,6 +681,128 @@ public function getProcessBalloons(): bool
         return $this->processBalloons;
     }
 
+    /**
+     * Set processAwards
+     *
+     * @param boolean $processAwards
+     *
+     * @return Contest
+     */
+    public function setProcessAwards($processAwards)
```

Comment:
We could add `bool` as typehint here (and in the return of the method below)

Reply:
Sure.